web/src/compose_actions.js

```diff
@@ -338,15 +338,16 @@ export function start(msg_type, opts) {
         compose_state.message_content(opts.content);
     }
 
-    if (opts.draft_id) {
-        $("#compose-textarea").data("draft-id", opts.draft_id);
-    }
-
     compose_state.set_message_type(msg_type);
 
     // Show either stream/topic fields or "You and" field.
     show_compose_box(msg_type, opts);
 
+    if (opts.draft_id) {
+        $("#compose-textarea").data("draft-id", opts.draft_id);
+        compose_validate.check_overflow_text();
+    }
+
```

Comment:
Relocated this portion of the code below because the ```show_compose_box(msg_type, opts)``` function was calling `clear_errors()`, and we did not want to interfere with the `compose_validate.check_overflow_text()` function.

Reply:
Sure!